functions/src/routers/groupRouter.ts

```diff
@@ -140,15 +140,13 @@ groupRouter.get("/:userId/groups", async (req, res) => {
       return res.status(404).json({ status: 404, error: "USERDATA NOT FOUND" });
 
     // 해당 유저 그룹에 그룹 정보가 있는지 확인
-    if (!userData.group)
-      return res
-        .status(403)
-        .json({ status: 403, error: "USERGROUP FORBIDDEN", uid });
+    if (!userData.groups)
+      return res.status(200).json({ status: 200, [] })
 
     // 유저의 그룹 정보 데이터 반환
     return res.json({
       status: "200",
-      data: userData.group, // Group ID
+      data: userData.groups, // Group ID
```

Comment:
```suggestion
      data: userData.groups || [], // Group ID
```
이런식으로 `||` 을 활용해서 정의가 안되어있으면 빈 배열을 출력하도록